src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml

```diff
@@ -180,18 +180,17 @@
     <!-- search control and include prerelease checkbox -->
     <Grid Grid.Row="2" Margin="20,0,0,0">
       <Grid.ColumnDefinitions>
-        <ColumnDefinition Width="*" MinWidth="269"/>
-        <ColumnDefinition Width="0.01*" MinWidth="5" />
-        <ColumnDefinition Width="*" MinWidth="192" />
+        <ColumnDefinition Width="2*"/>
+        <ColumnDefinition MinWidth="192" />
       </Grid.ColumnDefinitions>
-
+      
```

Comment:
undo

Reply:
there are some locations where we do an extra line after the ColumnDefinitions, is there a style guide for this?

Reply:
An extra line is ok but the extra spaces don't need to be there.